pkg/publisher/s3.go

```diff
@@ -112,6 +114,11 @@ func upload(options *ObjectOptions) error {
 	}
 	if *options.endpoint != "" {
 		awsConfig.Endpoint = options.endpoint
+	}
+
+	if options.forcePathStyle != nil {
+		awsConfig.S3ForcePathStyle = aws.Bool(options.forcePathStyle)
+	} else {
 		awsConfig.S3ForcePathStyle = aws.Bool(true)
 	}
```

Comment:
This looks to be changing the logic flow, as `S3ForcePathStyle` should only be set if `*options.endpoint != ""` is also true
This was the original logic flow:
```
if *options.endpoint != "" {
	awsConfig.Endpoint = options.endpoint
	awsConfig.S3ForcePathStyle = aws.Bool(true)
}
```
Should this if-else be within the if-statement above it?

Reply:
I See.